test/e2e-apiserver-test/oam_application_test.go

```diff
@@ -108,7 +108,7 @@ var _ = Describe("Test oam application rest api", func() {
 			Policies:   app.Spec.Policies,
 			Workflow:   app.Spec.Workflow,
 		}
-		res := post(fmt.Sprintf("/v1/namespaces/%s/applications/%s?dryRun=All", namespace, appName), req)
```

Comment:
Why should we change this?

Reply:
An error is returned in a unit test. Refer to the latest code for modifications.

Reply:
Refer to https://github.com/kubevela/kubevela/pull/5252/files#diff-0144ec5ef7981247aa9d9934f55282f0779e8f08b16b3636602120cdc4307e9a.